src/stream.js

```diff
@@ -3,5 +3,97 @@
 const logger = require('./logger');
 
 module.exports.handler = async (event) => {
+	event = decodeEvent({event});
 	logger.debug({event}, 'Event received');
 };
+
+class Question {
+	static get dbKeys() {
+		return ['date', 'channel'];
+	}
+
+	get answers() {
+		return this._answers;
+	}
+
+	get channel() {
+		return this._channel;
+	}
+
+	get date() {
+		return this._date;
+	}
+
+	constructor(image) {
+		this._date = image.date.S;
+		this._channel = image.channel.S;
+		this._answers = [];
+		Object.keys(image).forEach(key => {
+			if (!Question.dbKeys.includes(key) && image[key] && image[key].M) {
+				this._answers.push({
+					name: key,
+					answer: image[key].M.answer && image[key].M.answer.BOOL,
+					timestamp: image[key].M.timestamp && image[key].M.timestamp.N,
+				});
+			}
+		});
+		this._answers.sort(this._compareAnswers);
```

Comment:
I don't think we need to sort . from the docs:
> For each item that is modified in a DynamoDB table, the stream records appear in the same sequence as the actual modifications to the item.

Reply:
I think we can be lazy and just choose the record with the highest index for a given item.

Reply:
But these are the answers of a dynamoDB item. It is sorting the answers since we will get all answers from an item?